azure-pipelines.yml

```diff
@@ -258,11 +242,16 @@ steps:
       inlineScript: |
         echo "Chosen AKS Cluster name"
         echo $(chosenclustername)
+        # Delete CRDs to clean up cluster
+        kubectl delete -f ./charts/azure-service-operator/crds/
+        # Remove Helm deployment
+        helm list -n $(OPERATOR_NAMESPACE)
+        helm delete aso -n $(OPERATOR_NAMESPACE)
         # Set tags to available for the selected cluster to put it back into the free pool
         echo "Setting tags back to free"
         az resource tag --tags 'freeforpipeline=true' -g $(AKS_CLUSTER_RG) -n $(chosenclustername) --resource-type Microsoft.ContainerService/managedClusters
       workingDirectory: '$(System.DefaultWorkingDirectory)'
-      failOnStandardError: true
+      failOnStandardError: false
```

Comment:
Why wouldn't we want to fail on standard error? Can you add a comment to explain?

Reply:
Since it’s a cleanup step that doesn’t impact the actual pipeline I changed that but now that I think of it I think it might be good to fail so we get feedback if things are failing. Will update that.

Reply:
Fixed this now @WilliamMortlMicrosoft